src/redux/reducers/experiments/experimentsLoaded.js

```diff
@@ -0,0 +1,27 @@
+const experimentsLoaded = (state, action) => {
+  const { experiments } = action.payload;
+
+  const newExperiments = experiments.reduce((acc, curr) => ({
```

Comment:
instead of doing shallow copies for each iteration of the reduce wouldn't it be better to just insert the new object in each iteration? I think it might be more readable and efficiente if we don't need to do the copies.

Reply:
Yup, it'd be better. Rewritten as suggested.